api/src/config/mod.rs

```diff
@@ -259,6 +266,18 @@ impl Config {
         }
     }
 
+    pub fn frontend_config(&self) -> Option<FrontendConfig> {
+        self.frontend.as_ref().cloned()
+    }
+
+    pub fn frontend_title(&self) -> String {
+        self.frontend_config()
+            .as_ref()
+            .and_then(|fe| fe.title())
+            .cloned()
+            .unwrap_or_else(|| "PREvant".to_string())
+    }
+
```

Comment:
See suggestion in the index route and comment regarding idiomatic Rust.

```suggestion
```

The code also avoids some clones.

Reply:
Done.